content/blog/2023/2023-01-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: "Jenkins December 2022 Newsletter"
+tags:
+- jenkins
+- newsletter
+- community
+- contribute
+authors:
+- alyssat
+opengraph:
+image: /images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png
+links:
+discourse: true
+---
+
+== December 2022
+
+image:/images/post-images/2022-10-13-jenkins-newsletter/centered-newsletter.png[Jenkins 2022 recap Newsletter]
+
+Welcome to the Jenkins Newsletter! This is a compilation of progress within the project, highlighted by Jenkins Special Interest Groups (SIGs) for 2022.
+
+Happy reading!
+
+Got Inspiration? We would love to highlight your cool Jenkins innovations. Share https://docs.google.com/forms/d/e/1FAIpQLScMCGOMtn2hGpfXsbyssGhVW1LwlW4LkXCIaKINKDQU2m6ieg/viewform[your story] and you could be in the next Jenkins newsletter.
+
+== Highlights
+
+* Welcome Dr. Ullrich Hafner, Alexander Brandes and Kevin Martens as new members to the governance board and documentation officer.
+
+== Governance Update
+
+Contributed by: https://github.com/MarkEWaite[Mark Waite]
+
+The Jenkins governance board thanks Ewelina Wilkosz and Gavin Mogan for their two years of service as board members.
+We’re so glad to have talented people on the Jenkins governance board.
+
+December 2022 brings two new members to the Jenkins governance board.
+
+We welcome Dr. Ullrich Hafner of the Department of Computer Science and Mathematics at the Munich University of Applied Sciences is a new member of the board.
+In his role as professor he tries to win new Jenkins contributors by letting students develop new features and test cases in their student projects and thesis.
+He has been an active contributor in the Jenkins project since 2007, mostly in the acceptance test harness and the static code analysis suite (which is now replaced by the Warnings Next Generation Plugin).
```

Comment:
```suggestion
We welcome Dr. Ullrich Hafner of the Department of Computer Science and Mathematics at the Munich University of Applied Sciences as a new member of the board.
In his role as professor, he tries to win new Jenkins contributors by letting students develop new features and test cases in their student projects and thesis.
He has been an active contributor in the Jenkins project since 2007, mostly in the acceptance test harness and the static code analysis suite (which is now replaced by the Warnings Next Generation Plugin).
```
small suggestion for punctuation